README.md

```diff
@@ -490,12 +490,13 @@ Please note that using regular expressions in `VIRTUAL_HOST` will always result
 
 ### Troubleshooting
 
-In case you can't access your VIRTUAL_HOST, set `DEBUG=true` in the client container's environment and have a look at the generated nginx configuration file `/etc/nginx/conf.d/default.conf`:
+If you can't access your `VIRTUAL_HOST`, inspect the generated `/etc/nginx/conf.d/default.conf` file:
 
 ```console
 docker exec <nginx-proxy-instance> cat /etc/nginx/conf.d/default.conf
 ```
```

Comment:
While we're at it, we should suggest to look a the full configuration and use `nginx -T` to get it rather than doing `cat` on a single file. That's what I end up asking most of the time on issues.
```suggestion
If you can't access your `VIRTUAL_HOST`, inspect the generated nginx configuration:

```console
docker exec <nginx-proxy-instance> nginx -T
```
```